.github/workflows/CODEOWNERS

```diff
@@ -0,0 +1,18 @@
+clang/ @erichkeane, @Fznamznon
+
+clang/include/clang/Driver/ @mdtoguchi @AGindinson
+clang/lib/Driver/ @mdtoguchi @AGindinson
+
+llvm-spirv/ @AlexeySotkin, @AlexeySachkov
+
+opencl-aot/ @dm-vodopyanov
+
+sycl/doc/extensions/ @mkinsner, @jbrodman
+
+sycl/doc/ @pvchupin, @kbobrovs
+
+sycl/ @romanovvlad
+
+xpti/ @tovinkere
```

Comment:
I would like to have a backup code owner for xpti and @andykaylor has graciously agreed to it.